src/operator/quantization/asymmetric_quantize_graph_pass.cc

```diff
@@ -36,8 +36,7 @@ using nnvm::NodeEntry;
 using nnvm::Graph;
 
 template <bool require_bias>
-static inline bool IsOneDNNFullyConnected(const ObjectPtr& n) {
-#if MXNET_USE_MKLDNN == 1
+bool IsOneDNNFullyConnected(const ObjectPtr& n) {
```

Comment:
Please do not remove "static". Otherwise not required function by other modules will be visible outside this module. IT potentially could increase linking time